UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -0,0 +1,1631 @@
+using System;
+using System.Collections.Generic;
+using Core.Editor.Attributes;
+using HealthV2;
+using Items;
+using Messages.Server.SoundMessages;
+using Mirror;
+using Objects;
+using Objects.Construction;
+using UnityEngine;
+using UnityEngine.Events;
+using Util;
+using Random = UnityEngine.Random;
+
+public class UniversalObjectPhysics : NetworkBehaviour, IRightClickable
+{
+	//TODO Push objects being Reset our client
```

Comment:
are all of these TODO still TODO?

Reply:
some of them aren't

Reply:
it is probably a better idea to move them into issues (each one), after we merged this